includes/class-wc-payments-order-service.php

```diff
@@ -520,6 +520,36 @@ public function get_charge_id_for_order( $order ) : string {
 		return $order->get_meta( self::CHARGE_ID_META_KEY, true );
 	}
 
+	/**
+	 * Get the dispute id from an order's notes if it exists.
```

Comment:
Ideally, we would have stored this as order meta, similar to `_charge_id`.

I considered introducing an order meta solution. However, it would not be a backwards-compatible solution, as any previous disputed orders will not have this meta.

Reply:
Added this in f1f7dcc